pipeline/publication.py

```diff
@@ -1440,7 +1440,12 @@ def add_issns(self, part_of):
         if part_of.body.get("meta"):
             if not self.body.get("meta"):
                 self.body["meta"] = []
-            self.body["meta"].append(part_of.body.get("meta"))
+            
+            meta = part_of.body.get("meta")
+            if isinstance(meta, list):
+                self.body["meta"] = meta
```

Comment:
This could drop previous "metas" though, and perhaps ought to be `self.body["meta"] += meta` (or `.extend`).